WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceView.swift

```diff
@@ -265,7 +265,6 @@ private extension StorePerformanceView {
         if let chartViewModel = viewModel.chartViewModel {
             VStack {
                 StoreStatsChart(viewModel: chartViewModel) { selectedIndex in
-                    viewModel.trackInteraction()
```

Comment:
I wonder why this line of code has been removed. We do not want to track the interaction anymore?

Reply:
I left this comment in the [commit](https://github.com/woocommerce/woocommerce-ios/pull/14906/commits/b0e48674362db1d7a42d15665bed78992fcc7518) description but forgot to add it in this PR: This is already handled in didSelectStatsInterval with debounce logic to avoid emitting too many events.